pkg/config/templates/rancherd-22-addons.yaml

```diff
@@ -24,9 +28,176 @@ resources:
       repo: http://harvester-cluster-repo.cattle-system.svc/charts
       version: "0.2.4-rc2"
       chart: harvester-pcidevices-controller
+      {{- if and .Addons .Addons.harvester_pcidevices_controller }}
+      enabled: {{ .Addons.harvester_pcidevices_controller.Enabled }}
+      {{- else }}
       enabled: false
+      {{- end }}
       valuesContent: |
         image:
           tag: v0.2.4-rc2
         fullnameOverride: harvester-pcidevices-controller
-
+  - apiVersion: harvesterhci.io/v1beta1
+    kind: Addon
+    metadata:
+      name: rancher-logging
+      namespace: cattle-logging-system
+    spec:
+      repo: http://harvester-cluster-repo.cattle-system.svc/charts
+      version: "100.1.3+up3.17.7"
+      chart: rancher-logging
+      {{- if and .Addons .Addons.rancher_logging }}
+      enabled: {{ .Addons.rancher_logging.Enabled }}
+      {{- else }}
+      enabled: false
+      {{- end }}
+      valuesContent: |
+        images:
+          fluentbit:
+            tag: 1.9.5 # replace the default 1.9.3, which has bug in systemd log
+          fluentbit_debug:
+            tag: 1.9.5-debug
```

Comment:
Is this already added in the air-gap image list?

Reply:
we should add the debug image. the 1.9.5 image already exists in the rancher-images.txt